sources/ci/build-and-test.md

```diff
@@ -104,3 +105,18 @@ build:
 
 - The `on_failure` section will not be executed for timed out, unstable builds.
 - This section is also not executed if the build fails because code coverage does not meet the minimum threshold value.
+## always
+
+The `always` section is provided for any actions you want to take irrespective of your CI succeeds or fails. The commands in this section are executed no matter whether `ci` sections exits with 0 or not.
```

Comment:
The `always` section is provided for any actions you want to take whether your CI succeeds or fails. These commands will run even if your `ci` section exits with an error.